config/schedule.rb

```diff
@@ -1,3 +1,6 @@
+# Force manual loading of rails application to get all env variables from dotenv-rails when running whenever cmd
+require File.expand_path('../environment',  __FILE__)
```

Comment:
I thought that we may need to do this. It's a shame to load so much code just for some config variables but at least this command is only used once per deploy.

Reply:
Yes really bad in term of performance but as you say it's deploy time not execution time... the nice part is that in terms of code it's really lightweight.

Reply:
I wonder if it's even needed here, considering we effectively removed it months ago. At the same time we moved some other jobs into sidekiq scheduler.

If we're just scheduling some rake tasks, I think when those tasks actually run they will pick up the env vars as needed...? Know what I mean?

Reply:
I agree with you for `MAILTO` env var, but backups are scheduled only if  S3_BACKUPS_BUCKET is defined so this one seems mandatory.

Reply:
> backups are scheduled only if S3_BACKUPS_BUCKET is defined so this one seems mandatory.

I suspect in that case the rake task will just do nothing if there is no bucket, so the outcome is effectively the same (maybe with a tiny bit of inefficiency).